plugins/Goals/Reports/Get.php

```diff
@@ -182,6 +184,17 @@ public function configureView(ViewDataTable $view)
                 }
             }
         } else if ($view->isViewDataTableId(Evolution::ID)) {
+            if (!empty($idSite) && Piwik::isUserHasAdminAccess($idSite)) {
+                $view->config->title_edit_entity_url = 'index.php' . Url::getCurrentQueryStringWithParametersModified(array(
+                    'module' => 'Goals',
+                    'action' => 'manage',
```

Comment:
In 3.x they are not clickable when exported so shouldn't be a problem and don't think needs fixing in 2.x or even can be easily fixed